src/Host.h

```diff
@@ -591,40 +591,40 @@ class Host : public QObject
     // used - the previously used value of "yyyy-MM-dd#hh-mm-ss" has been
     // changed to "yyyy-MM-dd#HH-mm-ss" and is set as a default in the
     // constructor:
```

Comment:
Keep "why" comment from .cpp but remove outdated reference to constructor:
```suggestion
    // used - the previously used value of "yyyy-MM-dd#hh-mm-ss" has been
    // changed to "yyyy-MM-dd#HH-mm-ss" as we always want a 24-hour clock.
```